src/deadline/houdini_submitter/python/deadline_cloud_for_houdini/submitter.py

```diff
@@ -66,10 +72,28 @@ def _get_steps(node: hou.Node):
     wedged_steps = _get_wedge_steps(node)
     if wedged_steps is not None:
         # valid wedged network detected
-        return wedged_steps
+        rop_steps = wedged_steps
     else:
         # standard network
-        return _get_rop_steps(node)
+        rop_steps = _get_rop_steps(node)
+
+    if not separate_steps:
+        # render the node connected to the deadline cloud node
+        # and all its input nodes. The opposite of splitting it
+        # up each node by step
+
+        try:
+            connected_node = rop_steps[-1]
+        except IndexError:
+            return []
```

Comment:
nit: it would be better to raise an exception here (otherwise an exception will be raised down the chain), but based on the current code, this statement is actually unreachable, so not blocking